App.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link
+} from "react-router-dom";
+import HomePage from "./homepage.jsx";
+import PizzaPage from './pizza';
+import SalmonPage from './salmon';
+import PiePage from './pie';
+import AboutMe from './about';
+import {
+  useEffect,
+  useState,
+} from "react"
+
+export default function App() {
+  let [recipes, setRecipes] = useState([]);
+
+  useEffect(() => {
+    const loadRecipes = async () => {
+      let res = await fetch('http://localhost:3001/api/recipe', {mode: "no-cors"})
+      setRecipes(await res.json())
```

Comment:
The easiest fix for cors errors is to actually add the following lines into your index.js file on the backend. (I've never seen the approach you are using and if it works it works)

```
app.use((req, res, next) => {
  res.header("Access-Control-Allow-Origin", "*");
  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
  res.header('Access-Control-Allow-Methods', 'GET,POST,OPTIONS,DELETE,PUT');
  next();
});
```

Reply:
also you would want to add those lines above where your routes are called